src/test/java/org/sagebionetworks/bridge/services/FileServiceTest.java

```diff
@@ -0,0 +1,221 @@
+package org.sagebionetworks.bridge.services;
+
+import static org.sagebionetworks.bridge.TestConstants.GUID;
+import static org.sagebionetworks.bridge.TestConstants.TEST_STUDY;
+import static org.sagebionetworks.bridge.TestConstants.TEST_STUDY_IDENTIFIER;
+import static org.sagebionetworks.bridge.TestConstants.TIMESTAMP;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertSame;
+import static org.testng.Assert.assertTrue;
+
+import java.util.Optional;
+
+import org.joda.time.DateTime;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import org.sagebionetworks.bridge.dao.FileMetadataDao;
+import org.sagebionetworks.bridge.exceptions.BadRequestException;
+import org.sagebionetworks.bridge.exceptions.EntityNotFoundException;
+import org.sagebionetworks.bridge.exceptions.InvalidEntityException;
+import org.sagebionetworks.bridge.models.files.FileMetadata;
+
+public class FileServiceTest extends Mockito {
+
+    private static final String NAME = "oneName";
+    
+    @Spy
+    FileMetadataDao mockFileDao;
```

Comment:
Did you mean Mock? Is there any situation where we want to call through to the actual mockFileDao?

Reply:
Yes this should be a mock.